### PR TITLE
Convert MeanData to ConstSummary

### PR DESCRIPTION
**Pull request by songy23**
Closes https://github.com/census-instrumentation/opencensus-go/issues/363.

### CODE REVIEW

**Comment by rakyll** [comment]
The summary type is not going to work as expected without the quantiles. The users might get confused seeing no percentiles uploaded with the data. Can we document this in the exporter?

**Comment by songy23** [comment]
Sure, I added some document for `Collect()`.

File: exporter/prometheus/prometheus.go [hunk 1]
```diff
@@ -196,6 +196,12 @@ func (c *collector) Describe(ch chan<- *prometheus.Desc) {
 // and delivers them as Prometheus Metrics.
 // Collect is invoked everytime a prometheus.Gatherer is run
 // for example when the HTTP endpoint is invoked by Prometheus.
+// Each OpenCensus AggregationData will be converted to
```
Comment (rakyll) [inline exporter/prometheus/prometheus.go]: This new comments section is not readable by the user given collector is not exported, we can add it to the Export's godoc.

Comment (songy23) [reply exporter/prometheus/prometheus.go]: Yeah just realized that - I've moved it to ExportView().

**Comment by songy23** [comment]
Thanks!

**Pull request merged**